packages/sanity/src/core/studio/components/navbar/search/components/filters/filter/inputs/date/dateIncludeTimeFooter/DateIncludeTimeFooter.tsx

```diff
@@ -0,0 +1,22 @@
+import {Flex, Stack, Switch, Text} from '@sanity/ui'
+import React from 'react'
+
+const INCLUDE_TIME_LABEL = 'Include time'
+
+interface DateIncludeTimeFooterProps {
+  onChange: () => void
+  value: boolean
+}
+
+export function DateIncludeTimeFooter({onChange, value}: DateIncludeTimeFooterProps) {
+  return (
+    <Flex align="center" gap={2} justify="flex-end" marginTop={1}>
+      <Stack>
+        <Text muted onClick={onChange} size={1} style={{cursor: 'default'}} weight="medium">
```

Comment:
Nitpick: Prefer styled-component instead of inline styling, or move the inline styling object outside the component scope!